gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapperTest.java

```diff
@@ -90,6 +96,10 @@ public class MedicationRequestMapperTest {
             )
         )
     );
+    private static final Extension MEDICATION_STATEMENT_LAST_ISSUE_DATE_EXTENSION = new Extension(
+        "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationStatementLastIssueDate-1",
+        DateFormatUtil.parseToDateTimeType("20240726")
+    );
```

Comment:
Looks like this can just be part of the `buildMedicationStatement` method, only place it's being used!

Reply:
inlined this :)